src/components/learner-credit-management/BudgetDetailActivityTabContents.jsx

```diff
@@ -27,6 +29,10 @@ const BudgetDetailActivityTabContents = ({ enterpriseUUID, enterpriseFeatures })
     );
   }
 
+  if (!isTopDownAssignmentEnabled || !subsidyAccessPolicy?.isAssignable) {
```

Comment:
[context] Only renders the "Spent" table under the following situations:

* Top-down assignment feature flag is disabled
* There is no subsidy access policy (i.e., viewing an enterprise offer).
* The subsidy access policy is not assignable (i.e., it may be per-learner spent limit instead).

This helps prevent the "No budget activity yet? Assign a course!" empty state from showing incorrectly when not viewing an assignable budget or top-down assignment feature is not yet enabled.